flutter/lib/src/integrations/load_contexts_integration.dart

```diff
@@ -139,13 +140,27 @@ class _LoadContextsIntegrationEventProcessor extends EventProcessor {
       final breadcrumbsList = infos['breadcrumbs'] as List?;
       if (breadcrumbsList != null && breadcrumbsList.isNotEmpty) {
         final breadcrumbs = event.breadcrumbs ?? [];
+        final breadcrumbsJson =
+            breadcrumbs.map((breadcrumb) => breadcrumb.toJson());
+
         final newBreadcrumbs =
             List<Map<dynamic, dynamic>>.from(breadcrumbsList);
 
         for (final breadcrumb in newBreadcrumbs) {
-          final newBreadcrumb = Map<String, dynamic>.from(breadcrumb);
-          final crumb = Breadcrumb.fromJson(newBreadcrumb);
-          breadcrumbs.add(crumb);
+          final newBreadcrumbJson = Map<String, dynamic>.from(breadcrumb);
+
+          var containsDuplicate = false;
+          for (final breadcrumbJson in breadcrumbsJson) {
+            if (mapEquals(newBreadcrumbJson, breadcrumbJson)) {
+              containsDuplicate = true;
+              break;
+            }
+          }
+
+          if (!containsDuplicate) {
+            final newBreadcrumb = Breadcrumb.fromJson(newBreadcrumbJson);
+            breadcrumbs.add(newBreadcrumb);
+          }
```

Comment:
IIRC @krystofwoldrich and I talked about this already.
If `scopeSync` is enabled, use the crumbs from the bridge so you don't need to duplicate anything (it contains all the crumbs in the right order already).
If `scopeSync` is disabled, use the crumbs from the Dart scope.

Reply:
https://github.com/getsentry/sentry-react-native/blob/1542a6003154377ad611e0f0eca8902d3962fdef/src/js/integrations/devicecontext.ts#L92-L97

the RN SDK does not have a `enableScopeSync`, but we do.